docs/source/quickstart.mdx

```diff
@@ -74,6 +78,47 @@ The example files located in `graphql/TheSpaceDevs/` include:
 
     See the [command reference](/apollo-mcp-server/command-reference#config-options) for a list of available configuration options. 
 
+### Docker deployment
+
+1. Modify the configuration YAML for the example code to the following:
+    ```yaml graphql/TheSpaceDevs/config.yaml {7, 10}
+    endpoint: https://thespacedevs-production.up.railway.app/
+    transport:
+      type: streamable_http
+    operations:
+      source: local
+      paths:
+        - /data/operations
+    schema:
+      source: local
+      path: /data/api.graphql
+    overrides:
+      mutation_mode: all
+    introspection:
+      execute:
+        enabled: true
+      introspect:
+        enabled: true
+      search:
+        enabled: true
+    ```
+
+Then, from the root directory of your local repo, run this Docker command to start a local graph with an MCP Server:
```

Comment:
```suggestion
1. In a new terminal, from the root directory of your local repo, run this Docker command to start a local graph with the MCP Server:
```
Continue with the steps, adding clarification

Reply:
This will probably cause trouble further down the article and I didn't know the best way to deal with it, since there's only one step for running with Rover, but two steps for Docker. What would you recommend for differentiating? Maybe we add another heading for checking the MCP server using MCP Inspector?

Reply:
As we discussed last week, let's add a new step to handle the MCP Inspector step!